### PR TITLE
Fixed a bug in the related links.

### PR DESCRIPTION
**Pull request by marceloyuela**
For some reason all links in the cmdlets start with ( and end with ), throwing off the linking of the website.
This PR fixes that.

### CODE REVIEW

File: Tools/GenerateWebsiteData.ps1 [hunk 1]
```diff
@@ -56,7 +56,8 @@ function getLinks ($docObj) {
     $relatedLinks = $docObj.relatedLinks
     $links = @{}
     ForEach ($link in $relatedLinks.navigationLink) {
-        $links.Add($link.linkText,$link.uri)
+        $uri = $link.uri.replace("(","").replace(")","")
```
Comment (ILMTitan) [inline Tools/GenerateWebsiteData.ps1]: Should this be something like trimStart and trimEnd?

**Comment by ILMTitan** [comment]
I'm sorry I forgot to mention that. We do that because it is suggested by the xmldoc2cmdletdoc tool we use. I am a little concerned that this could replace an intentional parenthesis. Does javascript have a TrimStart and TrimEnd?

**Comment by marceloyuela** [comment]
(it was powershell so yes)

**Comment by ILMTitan** [comment]
LGTM

**Pull request merged**